### PR TITLE
Abstract Routing

### PR DESCRIPTION
**Pull request by TheBurchLog**
Utilizing pointers to the module functions that will invoke the action requested by the Forward Request.

### CODE REVIEW

File: src/app/beer_garden/router.py [hunk 1]
```diff
@@ -18,56 +17,178 @@
...
+    :return: Boolean response on if the object should be forwarded
+    """
+    # if forward.target_garden_name is None:
+    #     for arg in forward.args:
+    #         if isinstance(arg, (brewtils.models.Request, brewtils.models.RequestTemplate)):
+    #             forward.target_garden_name = get_system_mapping(
+    #                 name_space=arg.namespace,
+    #                 system=arg.system,
+    #                 version=arg.system_version,
+    #             )
```
Comment (hazmat345) [inline src/app/beer_garden/router.py]: Block-commented code needs to be removed before merging.

Comment (TheBurchLog) [reply src/app/beer_garden/router.py]: I can clean this up. I commented this to ensure it didn't run while testing. This should be functional after we get the plug-in registration working to make to `namespaces` and `garden_names`

**Comment by TheBurchLog** [comment]
Changing the name to something like `Operation` sounds fine with me. The name `Forward` is not intuitive to what is actually occurring.

I just updated my PyCharm to use Google for docstrings. So I can work on converting them over to that format. By hey, I actually wrote them this time :)

**Comment by hazmat345** [comment]
Hahahah, making progress :smile:

File: src/app/beer_garden/api/http/__init__.py [hunk 2]
```diff
@@ -202,6 +203,7 @@ def _setup_tornado_app():
         # These are a little special - unpublished but still versioned
         # The swagger spec
         (rf"{prefix}api/v1/spec/?", misc.SpecHandler),
+        (rf"{prefix}api/v1/forward/?", v1.forward.ForwardAPI),
```
Comment (hazmat345) [inline src/app/beer_garden/api/http/__init__.py]: :+1:

**Pull request merged**